app/src/main/java/org/wikipedia/search/PrefixSearchResponse.kt

```diff
@@ -0,0 +1,29 @@
+package org.wikipedia.search
+
+import com.google.gson.annotations.SerializedName
+import org.wikipedia.dataclient.mwapi.MwQueryResponse
+
+class PrefixSearchResponse : MwQueryResponse() {
+    @SerializedName("searchinfo")
+    private val searchInfo: SearchInfo? = null
+    private val search: Search? = null
```

Comment:
I feel we should leave the unused response attributes just for completion.. Lmk

Reply:
agree